modules/microshift-greenboot-create-health-check-script.adoc

```diff
@@ -7,11 +7,11 @@
 [id="microshift-greenboot-app-health-check-script_{context}"]
 = How to create a health check script for your application
 
-You can create workload or application health check scripts in the text editor of your choice using the example in this documentation. Save the scripts in the `/etc/greenboot/check/required.d` directory. When a script in the `/etc/greenboot/check/required.d` directory exits with an error, greenboot triggers a reboot in an attempt to heal the system.
+You can create workload or application health check scripts in the text editor of your choice using the "Example workload health check script" as a template. Save the scripts in the `/usr/lib/greenboot/check/required.d` directory. When a script in the `/usr/lib/greenboot/check/required.d` directory exits with an error, greenboot triggers a reboot in an attempt to heal the system.
```

Comment:
Here's the thing. That path is read-only on the ostree systems. If someone wants their script to reside there, it should be packaged and shipped via RPM.

user can use their favorite editor to create a script in `/etc/greenboot` no problem (with sudo)